src/pages/Autolending/PartnerDelRateDropdown.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div>
+		<h3 class="filter-title">
+			Partner delinquency rate
+		</h3>
+		<kv-dropdown-rounded v-model="delinquencyRate">
+			<option value="default">
```

Comment:
If you use numbers instead of strings for the option values (e.g. `0` for default, `5` for 5% or less, etc.) Then you won't need any of those if/else statements that are mapping the number values to string values.

Reply:
Alright, I'll go through and make those changes in this branch and the other branches I've created using strings.